packages/ssr-web/deployment/fastboot-server.js

```diff
@@ -1,6 +1,9 @@
 /* eslint-disable no-undef */
 const FastBootAppServer = require('fastboot-app-server');
 const fetch = require('node-fetch');
+const Sentry = require('@sentry/node');
+
+Sentry.init({ dsn: process.env.SSR_WEB_SERVER_SENTRY_DSN });
```

Comment:
Can we add an `environment` property, or the corresponding Sentry environment variable? The deployment step doesn't seem like it will get the `SENTRY_ENVIRONMENT` variable from the build step.

Reply:
Added!